addons/mail/static/src/components/composer_text_input/composer_text_input.js

```diff
@@ -217,6 +217,7 @@ export class ComposerTextInput extends Component {
         }
         this._textareaLastInputValue = this._textareaRef.el.value;
         this._updateHeight();
+        this.composerView.composer.removeLinkPreview();
```

Comment:
Since this is just an extra I add to "autoremove" the attachment when the user delete the link from the composer, I have remove that feature completely to avoid deal with this.